### ISSUE TITLE
Failing to compile since move of std:io to std::old_io

### ISSUE THREAD

**Issue by rgladwell** [lfairy/maud#6]
Maud is failing to compile with latest Rust nightly I think because of the [rename of the `std::io` to `std::old_io`](http://doc.rust-lang.org/std/old_io/):

```
/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/maud-0.1.1/src/lib.rs:160:15: 160:22 error: unresolved import `std::io::IoError`. Could not find `io` in `std`

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/maud-0.1.1/src/lib.rs:160 use std::io::{IoError, IoErrorKind, IoResult};

^~~~~~~

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/maud-0.1.1/src/lib.rs:160:24: 160:35 error: unresolved import `std::io::IoErrorKind`. Could not find `io` in `std`

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/maud-0.1.1/src/lib.rs:160 use std::io::{IoError, IoErrorKind, IoResult};

^~~~~~~~~~~

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/maud-0.1.1/src/lib.rs:160:37: 160:45 error: unresolved import `std::io::IoResult`. Could not find `io` in `std`

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/maud-0.1.1/src/lib.rs:160 use std::io::{IoError, IoErrorKind, IoResult};

^~~~~~~~
```

**Comment by lfairy** [comment]
Fixed in master and v0.1.2 -- thanks.

**Comment by rgladwell** [comment]
Thanks for fixing so quickly :+1:
